data/devices/meizu.yml

```diff
@@ -0,0 +1,34 @@
+ ---
```

Comment:
The build failed because there's an extra space at the beginning of the file. Can you please remove it?

```
meizu.yml: Failed to convert file: yaml-cpp: error at line 2, column 7: illegal map value
```